feature/security/gnsi/certz/tests/README.md

```diff
@@ -0,0 +1,60 @@
+# gNSI TLS Authentication Testing
+
+## Summary
+Test gRPC TLS authentication under various conditions created using
+combinations of different security credentials and artifacts manageable using
+the gNSI protocol, including device certificates, Trust-Bundle (TB),
+Certificate Revocation List (CRL) and Authentication Policy.
+
+## Authentication Policy Enforcement
+
+Test device enforcement of
+[`gnsi.certz.v1.AuthenticationPolicy`](https://github.com/openconfig/gnsi/blob/main/certz/certz.proto)
```

Comment:
@danielywong can you help provide a link to the doc/spec about the schema of the policy? Thanks.